PhpAmqpLib/Message/AMQPMessage.php

```diff
@@ -39,9 +39,9 @@ class AMQPMessage extends GenericContent
 
     /**
      * @param string $body
-     * @param null $properties
+     * @param array $properties
      */
-    public function __construct($body = '', $properties = null)
+    public function __construct($body = '', $properties = array())
```

Comment:
You're right... I reverted it and will add the change back into #201 since this PR should be only related to CS.